truffle.js

```diff
@@ -50,7 +50,7 @@ module.exports = {
   },
   compilers: {
     solc: {
-      version: "0.5.5",
+      version: "0.5.9",
```

Comment:
this change broke our coverage tool.
need to update `coverage-prov.js` to the same version, run `yarn coverage` to see the coverage is running.
(note that if you run coverage and then change the version, you need to `rm -rf .0x-artifacts` before running coverage again)